### PR TITLE
Add metadata files

### PR DESCRIPTION
**Pull request by anxolin**
# Summary
This PR removes the `APP_ID` setting and adds `METADATA_DIGEST_HEX`.

This is the hexadecimal digest of the content of a JSON file uploaded to IPFS with the metadata you want to attach by default to all orders.

This metadata will allow to give a better information about the context in which the order was created.

## Uploaded CowSwap and Balancer json files to IPFS
For this PR, it was added a JSON file with the metadata for Balancer and CowSwap which could be used to segment where the trades comes from.

For example, CowSwap uploaded the file https://cloudflare-ipfs.com/ipfs/QmX3raf5oWVB3VK8RE6VTMrUGWzCY8Fykpf9AVLXvxT954 
which has the hexadecimal digest `0x816BBB5787AA2B72E2DE870A4E935A49A97305DE32E637B8126D07BEE10D03AB` (See 
[CID Explorer](https://cid.ipfs.io/#QmX3raf5oWVB3VK8RE6VTMrUGWzCY8Fykpf9AVLXvxT954) for more details).

## Readme
This PR also updates the readme to reflect this

### CODE REVIEW

File: src/assets/metadata/Balancer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "version": "0.1.0",
+  "appCode": "Balancer",
```
Comment (alfetopito) [inline src/assets/metadata/Balancer.json:3]: I don't think we should include the balance file in the cowswap UI

File: src/assets/metadata/ipfs_hashes.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "Balancer": {
```
Comment (alfetopito) [inline src/assets/metadata/ipfs_hashes.json:2]: Not sure the cowswap repo is the right place for storing this info.
Maybe a gist would make more sense, to de-couple from one UI?

File: src/custom/constants/index.ts [hunk 3]
```diff
@@ -22,7 +22,8 @@ export const PERCENTAGE_PRECISION = 2
 
 export const LONG_LOAD_THRESHOLD = 2000
 
-export const APP_ID = Number(process.env.REACT_APP_ID)
+export const METADATA_DIGEST_HEX =
```
Comment (alfetopito) [inline src/custom/constants/index.ts:25]: What about `APP_DATA_HASH`?

File: src/assets/metadata/Balancer.json (hunk 1 above)
Comment (anxolin) [reply src/assets/metadata/Balancer.json]: okokko, was just to make it easy for us, but a document would be better suited. Removing. Created also: https://docs.google.com/spreadsheets/d/16RzefdfpmEavl182h6iNEe7fRyAcaATCRYR0hMQ_734/edit#gid=0

File: src/custom/constants/index.ts (hunk 3 above)
Comment (anxolin) [reply src/custom/constants/index.ts]: I called digest, cause that was what IPFS inspector showed. Hex, is just so is clearly an hex and not base52 or other encoding.  Metadata, cause is order metadata. 

Anyways, using yours, not sure what is better, seams alright to me

**Review by alfetopito — APPROVED** [review]

**Pull request merged**